ganalytics.php

```diff
@@ -125,10 +139,7 @@ private function getTrackingSettings($trackingId, $objectName)
 
         $settings = [
           'trace-debug' =>  "window.ga_debug = {trace: true};",
-          'create'      => "{$objectName}('create', '{$trackingId}', {$cookie_config});",
-          'anonymize'   => "{$objectName}('set', 'anonymizeIp', true);",
-          'force-ssl'   => "{$objectName}('set', 'forceSSL', true);",
-          'send'        => "{$objectName}('send', 'pageview');"
```

Comment:
**Question:** This breaks backwards compatibility, doesn't it? If I currently use UA ID with these settings other than default, it might break the tracking, right?